client/src/client/workers/test-evaluator.js

```diff
@@ -57,6 +58,7 @@ self.onmessage = async e => {
   /* eslint-disable no-unused-vars */
   const { code = '' } = e.data;
   const assert = chai.assert;
+  const helpers = curriculumHelpers;
```

Comment:
```suggestion
  const __helpers = curriculumHelpers;
```
Any variables we declare here can clash with user defined code.  So, if a user declares `var helpers = {}` then tests relying on `curriculumHelpers` will fail.

While it is not a perfect solution, I think a slightly unusual name like `__helpers` is less likely to be created by users.

Reply:
I'm not sure this is true. I thought I gave that a test. But probably not a bad idea to make it more unique anyway. Yea, I could not overwrite or reassign the helpers with a quick test on a JS challenge.

Reply:
It does reassign them - the user's code is `eval`ed after `helpers` is defined.  You can test it like this:

1. Go to http://localhost:8000/learn/javascript-algorithms-and-data-structures/basic-javascript/replace-loops-using-recursion
2. enter the following 
```js
function sum(arr, n) {
  // Only change code below this line
  if(n <= 0) {
    return 0;
  } else {
    return sum(arr, n - 1) + arr[n - 1];
  }
  // Only change code above this line
}

var helpers = {};
```
3. run tests.
4. see `TypeError: helpers.removeJSComments is not a function` in the browser console.